lib/modules/manager/npm/extract/index.ts

```diff
@@ -1,3 +1,4 @@
+import { major } from '@renovatebot/ruby-semver';
```

Comment:
? 😉

Reply:
Sorry, vscode imported this automatically